platform/android/src/test/main.jni.cpp

```diff
@@ -48,54 +42,35 @@ struct Main {
 
 } // namespace
 
-// JNI Bindings to stub the android.util.Log implementation
-
-static jboolean isLoggable(JNIEnv* env, jni::jobject* clazz, jni::jstring* tag, jint level) {
-    return true;
-}
-
-static jint println_native(JNIEnv* env, jni::jobject* clazz, jint bufID, jint priority, jni::jstring* jtag, jni::jstring* jmessage) {
-    if (jtag == nullptr || jmessage == nullptr) {
-        return false;
-    }
-
-    std::string tag = jni::Make<std::string>(*env, jni::String(jtag));
-    std::string message = jni::Make<std::string>(*env, jni::String(jmessage));
-
-    return __android_log_print(priority, tag.c_str(), "%s", message.c_str());
-}
-
-static jint logger_entry_max_payload_native(JNIEnv* env, jni::jobject* clazz) {
-    return static_cast<jint>(4068);
-}
-
+// We're declaring the function, which is libandroid_runtime.so.
+// It is defined in AndroidRuntime.cpp:
+// https://github.com/android/platform_frameworks_base/blob/master/core/jni/AndroidRuntime.cpp
+// Once this symbol goes away, we'll have to revisit.
```

Comment:
If this symbol goes away, the Mapbox GL SDK will start failing, right? It would be worse than a build time failure -- existing installations might stop working after an OS or software upgrade.

If this is not a public API, let's find a different way.